cookiecutter/extensions.py

```diff
@@ -40,11 +40,26 @@ def __init__(self, environment: Environment) -> None:
         """Jinja2 Extension Constructor."""
         super().__init__(environment)
 
-        def random_ascii_string(length: int, punctuation: bool = False) -> str:
+        def random_ascii_string(
+            length: int, number: bool = False, punctuation: bool = False
```

Comment:
I think that `numbers` matches my expectations for this parameter name.

Changing this will cause a few places in this PR to change.

```suggestion
            length: int, numbers: bool = False, punctuation: bool = False
```